src/aws-cpp-sdk-core/include/aws/core/AmazonSerializableWebServiceRequest.h

```diff
@@ -39,7 +39,7 @@ namespace Aws
             AWS_UNREFERENCED_PARAM(AMZN_XML_CONTENT_TYPE);
         }
 
-        virtual ~AmazonSerializableWebServiceRequest() {}
```

Comment:
nit: space before braces. do we actually need braces in this case at all?

Reply:
no, there currently is no consistency with having or not having the empty {}

Reply:
should I remove all of them